data/mods/Item_Category_Overhaul/categories.json

```
@@ -0,0 +1,265 @@
+[
+  {
+    "id": "tools",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "UNSORTED TOOLS" },
```

Might work better than unsorted? Not sure.
```suggestion
    "name": { "str": "OTHER TOOLS" },
```